pygyro/advection/accelerated_advection_steps.py

```diff
@@ -52,141 +52,150 @@ def poloidal_advection_step_expl( f, dt, v, rPts, qPts,
+            endPts_k1_r[i, j] = rPts[j] + dthetaPhi_0[i, j]*multFactor
 
             # Handle theta boundary conditions
-            while (endPts_k1_q[i,j]<0):
-                endPts_k1_q[i,j]+=2*pi
-            while (endPts_k1_q[i,j]>2*pi):
-                endPts_k1_q[i,j]-=2*pi
+            while (endPts_k1_q[i, j] < 0):
+                endPts_k1_q[i, j] += 2*pi
+            while (endPts_k1_q[i, j] > 2*pi):
+                endPts_k1_q[i, j] -= 2*pi
 
-            if (not (endPts_k1_r[i,j]<rPts[0] or
-                     endPts_k1_r[i,j]>rMax)):
+            if (not (endPts_k1_r[i, j] < rPts[0] or
+                     endPts_k1_r[i, j] > rMax)):
                 # Add the new value of phi to the derivatives
                 # x^{n+1} = x^n + 0.5( f(x^n) + f(x^n + f(x^n)) )
                 #                               ^^^^^^^^^^^^^^^
-                drPhi_k[i,j]     = eval_spline_2d_scalar(endPts_k1_q[i,j],endPts_k1_r[i,j],
-                                                        kts1Phi, deg1Phi, kts2Phi, deg2Phi,
-                                                        coeffsPhi,0,1)
-                drPhi_k[i,j]     /= endPts_k1_r[i,j]
-
-                dthetaPhi_k[i,j] = eval_spline_2d_scalar(endPts_k1_q[i,j],endPts_k1_r[i,j],
-                                                        kts1Phi, deg1Phi, kts2Phi, deg2Phi,
-                                                        coeffsPhi,1,0)
-                dthetaPhi_k[i,j] /= endPts_k1_r[i,j]
+                drPhi_k[i, j] = eval_spline_2d_scalar(endPts_k1_q[i, j], endPts_k1_r[i, j],
+                                                      kts1Phi, deg1Phi, kts2Phi, deg2Phi,
+                                                      coeffsPhi, 0, 1)
+                drPhi_k[i, j] /= endPts_k1_r[i, j]
+
+                dthetaPhi_k[i, j] = eval_spline_2d_scalar(endPts_k1_q[i, j], endPts_k1_r[i, j],
+                                                          kts1Phi, deg1Phi, kts2Phi, deg2Phi,
+                                                          coeffsPhi, 1, 0)
+                dthetaPhi_k[i, j] /= endPts_k1_r[i, j]
             else:
-                drPhi_k[i,j]     = 0.0
-                dthetaPhi_k[i,j] = 0.0
+                drPhi_k[i, j] = 0.0
+                dthetaPhi_k[i, j] = 0.0
 
             # Step two of Heun method
             # x^{n+1} = x^n + 0.5( f(x^n) + f(x^n + f(x^n)) )
-            endPts_k2_q[i,j] = (qPts[i] - (drPhi_0[i,j]     + drPhi_k[i,j])*multFactor_half) % (2*pi)
-            if endPts_k2_q[i,j]<0: # Needed for C due to pyccel issue #854
-                endPts_k2_q[i,j] += 2*pi
-            endPts_k2_r[i,j] = rPts[j] + (dthetaPhi_0[i,j] + dthetaPhi_k[i,j])*multFactor_half
+            endPts_k2_q[i, j] = (
+                qPts[i] - (drPhi_0[i, j] + drPhi_k[i, j])*multFactor_half) % (2*pi)
+            if endPts_k2_q[i, j] < 0:  # Needed for C due to pyccel issue #854
+                endPts_k2_q[i, j] += 2*pi
```

Comment:
Obsolete comment. This should now be handled by the `%` operator

Reply:
same as below